spec/use_cases/slack/send_messages_spec.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "../../spec_helper"
```

Comment:
It might be better to put this in a `.rspec` file so then it'll automatically be required for each test.

Example: https://github.com/alphagov/email-alert-api/blob/master/.rspec

Reply:
THANK YOU! I vaguely wondered why it wasn't auto-required but didn't have time to look into it.